packages/core/src/credential/Credential.ts

```diff
@@ -292,118 +277,114 @@ export function fromClaim(
   return credential
 }
 
-export interface TrustPolicy {
-  /**
-   * If set to true, attestations issued directly by this identity are accepted.
-   */
-  isAttester?: boolean
-  /**
-   * If set to true, attestations linked to a delegation node which this identity has control over are accepted.
-   */
-  isDelegator?: boolean
+type VerifyOptions = {
+  ctype?: ICType
```

Comment:
I find it strange that CType isn't required. Is there a specific reason why?

Reply:
see previous comment